config/agent/agent.conf

```diff
@@ -21,14 +22,16 @@ ManagerHost=
 #ManagerPort=842
 
 #
-# SD Bus address used by hirte-agent to connect to hirte. See `man sd_bus_set_address` for its format.
-# Overrides any setting of ManagerHost or ManagerPort defined in the configuration file as well as the respective
-# CLI options.
+# SD Bus address used by hirte-agent to connect to hirte. See
```

Comment:
If you already break the lines why not move `See` to the next one? It will still be <80